customization/mail-templates/assign-enquiry-notification.html

```diff
@@ -42,8 +42,7 @@ <h1 style='color: #FFFFFF; font-size: 30px; font-weight: bold; line-height: 40px
                               <b style="font-size: 20px; line-height: 28px; color: #333;">Liebe_r <span
                                     data-th-text="${name_recipient}"></span>,</b>
                               <br /><br />
-                              <span data-th-text="${name_sender}"></span> hat dir <span
-                                 data-th-text="${name_user}"></span> als neue_n Ratsuchende_n zugewiesen.
+                              <span data-th-text="${name_sender}"></span> <span data-th-text="#{mail.label.has.assigned.you.new.adviceseeker}"/>
```

Comment:
I would use the mail key only for common stuff inside mails.
For this phrase which leads to this mail for example i would recommend mail.assign-enquiry-notification. ...